docker-compose.yml

```diff
@@ -0,0 +1,7 @@
+shipit:
+  build: ./
+  command: python kickoff-web.py -d sqlite:///update.db --username=admin --password=password --host=0.0.0.0 --port=5000 --verbose
```

Comment:
I threw my initial attempt at using MySQL in https://gist.github.com/Callek/c5ad4b68af3a4674d2aa but I don't have enough understanding of shipit atm to duplicate the `initdb` aspects of of equivelant balrog stuff